### ISSUE TITLE
week04: Going deeper description

### ISSUE THREAD

**Issue by NickVeld** [yandexdataschool/Practical_RL#153]
> categorical crossentropy (see here). 

* The link does not work.

> In ideal casae this totals to 2 .dot's, 1 softmax and 1 sigmoid

* casae
* For pytorch, no direct calls of .dot, thus "2 layers" as in previous lines is more prefferable in my mind

> Do not initialize weights with zeros (due to symmetry effects). A gaussian noize with small variance will do.

* Pytorch initializes weights with not only zeros by default without user attention

**Comment by justheuristic** [comment]
Thanks! fixed it
https://github.com/yandexdataschool/Practical_RL/commit/5fea91667e2781b34f1437fc922b9f58620df37a
